packages/runtime/lib/config.js

```diff
@@ -89,10 +89,20 @@ async function _transformConfig (configManager, args) {
     if (configManager._fixPaths && service.config) {
       service.config = pathResolve(service.path, service.config)
     }
+
+    if (service.config) {
+      const manager = new ConfigManager({ source: pathResolve(service.config) })
+      await manager.parse()
+      const serviceConfig = manager.current
+      const type = serviceConfig.$schema ? ConfigManager.matchKnownSchema(serviceConfig.$schema) : undefined
+      service.type = type
+    }
+
     service.entrypoint = service.id === config.entrypoint
     service.dependencies = []
     service.localServiceEnvVars = new Map()
     service.localUrl = `http://${service.id}.plt.local`
+    service.isPLTService = service.type === 'service' || service.type === 'db' || service.type === 'composer'
```

Comment:
```suggestion
    service.isPLTService = service.type === 'service' || service.type === 'db' || service.type === 'composer'
```

On second thought this would not work for derivative of service outside of this repo, such as ai-warp, etc. 

A good solution could be to add a property to  https://github.com/platformatic/platformatic/blob/8361e7944d837eed258ef8cdfc095aaf7986c8d2/packages/service/index.js#L186, which should accessible as the result of `loadConfig` from the store.

Alternatively, we can add it as a schema property in all schemas, setting it to `true` everywhere but Platformatic own services.

Reply:
I would prefer to add  `isPLTService` property like `configType` on `composer`, `db`, `service` so the PLT services are asserting that they are default PLT services, with telemetry managed by Platformatic. All the others with `isPLTService` falsy will use open telemetry instrumentation.